shub/config.py

```diff
@@ -131,6 +133,8 @@ def save(self, path=None):
             yml['apikeys'] = self.apikeys
             yml['version'] = self.version
             yml['stacks'] = self.stacks
+            if self.eggs:
+                yml['requirements'] = {'eggs': self.eggs}
```

Comment:
Hm, I'm not super-fond of having both `requirements` and `requirements_file`. Should we deprecate the `requirements_file` parameter and instead use a `file` string or `files` list under the `requirements` dict? Then again it sucks to already deprecate the quite fresh `requirements_file` parameter. (/cc @dangra)